src/Uno.UI/Microsoft/UI/Input/InputLightDismissAction.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace Microsoft.UI.Input
 {
-	public sealed class InputLightDismissAction : InputObject
+	public sealed partial class InputLightDismissAction : InputObject
```

Comment:
Same here

Reply:
Because those types are added by the 1.2 WinAppSDK, and they're now generated by the SyncGenerator.

Reply:
@jeromelaban I thought we persist sync generator output on disk?

Reply:
Not for winui, which is the goal of this particular PR. We'll eventually switch to WinUI for the original sources, but we're not there yet.